app/controllers/content_items_controller.rb

```diff
@@ -65,23 +65,43 @@ def redirect_canonical(content_item)
   end
 
   def can_view(item)
```

Comment:
I feel like we should a question mark to this method since it only returns a bool